README.md

```diff
@@ -206,7 +208,17 @@ https://user-images.githubusercontent.com/1991296/224442907-7693d4be-acaa-4e01-8
 
 ## Usage
 
-Here are the steps for the LLaMA-7B model.
+**Quickstart:** You will find prebuilt Windows binaries on the release page.
```

Comment:
If we are going to add this it should be a proper section, e.g. `### Quickstart`.